tests/conftest.py

```diff
@@ -1152,6 +1152,11 @@ def nodes_cpu_architecture(nodes):
     return utilities.infra.get_nodes_cpu_architecture(nodes=nodes)
 
 
+@pytest.fixture(scope="session")
+def cluster_architecture_is_390x(nodes_cpu_architecture):
+    return nodes_cpu_architecture == "s390x"
```

Comment:
this is just a wrapper to check `nodes_cpu_architecture == "s390x"`; used in one place, so please check where needed